docs/install_guides/installing-openvino-linux.md

```diff
@@ -42,20 +40,21 @@ Optimized for these processors:
 
 - Ubuntu 18.04.x long-term support (LTS), 64-bit
 - Ubuntu 20.04.0 long-term support (LTS), 64-bit
-- CentOS 7.6, 64-bit (for deployment only, not development)
-- For deployment on Red Hat* Enterprise Linux* 8.2, 64-bit, you can use the Intel® Distribution of OpenVINO™ toolkit runtime package that includes the Inference Engine core libraries, nGraph, OpenCV, Python bindings, and CPU and GPU plugins. The package is available as: 
+- For deployment on Red Hat* Enterprise Linux* 8.2, 64-bit, you can use the Intel® Distribution of OpenVINO™ toolkit runtime package that includes the Inference Engine core libraries, nGraph, Python bindings, and CPU and GPU plugins. The package is available as: 
```

Comment:
Starting 2022.1 Red Hat* Enterprise Linux* 8.2 will be part of installer besides other distribution: archive, PyPI, Docker Image. I think this clarification should be removed.